### DISCUSSION TITLE
Uncaught Error: [Package Error] "http" does not exist. (Imported by "@graphql-yoga/node")

### DISCUSSION THREAD

**Discussion by dagda1** [denoland/deno#15667] [Q&A]
I am trying to use [@graphql-yoga/node], but when using it via this entry in `import_map.json`:

```json
"@graphql-yoga/node": "https://cdn.skypack.dev/@graphql-yoga/node",
```

I get this error:

> Uncaught Error: [Package Error] "http" does not exist. (Imported by "@graphql-yoga/node").
throw new Error("[Package Error] \"http\" does not exist. (Imported by \"@graphql-yoga/node\").");

Is there anything that can be done to polyfill `http`, I've tried adding `--compat --unstable` but no cigar.

**Comment by bartlomieju** [comment]
`--compat` is no longer supported. Have you tried importing it using `npm:` specifier instead of using version from CDN?

**Comment by vicary** [comment]
In my module `fresh_graphql`, `@graphql-yoga/common` works for my example. Haven't tried `@graphql-yoga/node` but you may give it a try.

Here is an extract of my `import_map.json`

```json
{
  "imports": {
    "@graphql-yoga/common": "https://esm.sh/@graphql-yoga/common@2.12.12?external=graphql",
    "graphql": "https://esm.sh/graphql@16.6.0"
  }
}
```
